src/datafactory/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.2.0'
+VERSION = '0.1.0'
```

Comment:
Can version decrease?

Reply:
Caused by codegen change. The `VERSION` here will now be overwritten by `from azext_datafactory.manual.version import VERSION` and that is changed to `0.3.0` manually.